openmc/lattice.py

```diff
@@ -408,6 +432,128 @@ def get_cell_instance(self, path, distribcell_index):
 
         return offset
 
+    def find_element(self, point):
+        """Determine index of lattice element and local coordinates for a point
+
+        Parameters
+        ----------
+        point : Iterable of float
+            Cartesian coordinates of point
+
+        Returns
+        -------
+        2- or 3-tuple of int
+            A tuple of the corresponding (x,y,z) lattice element indices
+        3-tuple of float
+            Carestian coordinates of the point in the corresponding lattice
+            element coordinate system
+
+        """
+        ix = floor((point[0] - self._lower_left[0])/self._pitch[0])
```

Comment:
Sorry, got a little bit overzealous in my optimization efforts. Turns out using the properties really doesn't impact performance from what I can tell so I'll switch back.